config/Jamfile.v2

```diff
@@ -34,6 +41,7 @@ project : requirements
    <toolset>msvc:<warnings>all
    <toolset>gcc:<cxxflags>-Wall
    <toolset>gcc:<cxxflags>-Wextra
+   <toolset>gcc:<cxxflags>-fext-numeric-literals  # enable suffix Q for float128.
```

Comment:
No, we *DO NOT* support -fext-numeric-literals only -std=gnu++XX - this puts gcc into an odd state where neither ourselves nor the C++ std lib can determine whether `__float128` is fully supported or not.  Plus the whole point of the configuration checks is to determine whether or not __float128 is fully supported -  we shouldn't be second guessing this, just determine whether the current compilation mode is OK or not.